mellon_create_metadata.sh

```diff
@@ -1,92 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@itewk was this file moved?  Looks like it's suppose to be in a subdir now.

Reply:
i removed it from this repository as it was almost an exact clone of https://github.com/Uninett/mod_auth_mellon/blob/master/mellon_create_metadata.sh but with a change to the file name generation. It felt odd/wrong to try and maintain a fork of that script and dealing with drift from https://github.com/Uninett/mod_auth_mellon/blob/master/mellon_create_metadata.sh so I updated the read me to have the user clone that repo as well to access that script so this repo doesn't have to deal with maintaining a copy of it.

If you prefer I can add it back, though I think I would app back from adding an exact clone of the one from https://github.com/Uninett/mod_auth_mellon/blob/master/mellon_create_metadata.sh save adding a comment saying that that it is a clone from there.

let me know.

Reply:
I suspect I had cloned it originally since we were intending this project to be something we officially supported in OpenShift (and therefore wouldn't want to depend on a third party website).  However, since this is more of a reference architecture / borderline documentation I think your approach is better now.